lib/palindrome_check.rb

```diff
@@ -1,7 +1,21 @@
-# A method to check if the input string is a palindrome.
-# Return true if the string is a palindrome. Return false otherwise.
-# Time complexity: ?
-# Space complexity: ?
 def palindrome_check(my_phrase)
-  raise NotImplementedError
+  if my_phrase == nil
+    return false
+  end
+  i = 0
```

Comment:
Nit: use more descriptive variable names?

This can be helpful in whiteboard interviews, as it helps you (and your interviewer) keep track of which is which. :smile: